UI-Session-1-Assignment/index.html

```diff
@@ -0,0 +1,197 @@
+                    <td><label>Vehicle Type</label></td>
+                    <td><select name="typeOfVehicle">
+                            <option disabled selected>--Select-Type--</option>
+                            <option value="Cycle">Cycle</option>
+                            <option value="MotorCycle">MotorCycle</option>
+                            <option value="Four Wheelers">Four Wheelers</option>
+                        </select></td>
+                </tr>
+                <tr>
+                    <td><label>Vehicle Number</label></td>
+                    <td><input name="vehicleNumber" type="text" placeholder="Vehicle Number" size="60%"
+                            pattern="^[A-Z]{2}[ -][0-9]{1,2}(?: [A-Z])?(?: [A-Z]*)? [0-9]{4}$" required></td>
+                </tr>
+                <tr>
+                    <td><label>Employee Email</label></td>
+                    <td><input name="email" type="text" size="60%" required>
+                    </td>
+                </tr>
+                <tr>
+                    <td><label>Identification</label></td>
+                    <td><textarea name="vehicleIdentification" placeholder="Identification" cols="60"
+                            required></textarea></td>
+                </tr>
+            </table>
+            <input name="addVehicleButton" type="submit" value="Add">
+        </form>
+    </div>
+    <br>
+    <!-- Div for feedback form -->
+    <div id="feedback_form" align="center">
+        <h2>
+            Get In Touch
+        </h2>
+        <form action="" method="get">
+            <table align="center" cellpadding="10">
+                <tr>
+                    <td><label>Join Date</label></td>
+                    <td><input type="text" placeholder="Join Date" id="join_date" size="60%" required></td>
+                </tr>
+                <tr>
+                    <td><label>Email</label></td>
+                    <td><input type="text" placeholder="Email" id="email" size="60%" required></td>
+                </tr>
+                <tr>
+                    <td><label>Subject</label></td>
+                    <td><input type="text" placeholder="Subject" id="subject" size="60%" required></td>
+                </tr>
+                <tr>
+                    <td><label>Message</label></td>
+                    <td><textarea rows="4" cols="61" placeholder="Message" id="message" required></textarea> </td>
+                </tr>
+            </table>
+            <br>
+            <input type="submit" value="Send Email" id="feedback_submission_button" colspan="2">
+        </form>
+    </div>
+    <br>
+    <!-- Div for price details section -->
+    <div id="price_details" align="center">
+        <h2>Pricing Details</h2>
+        <div id="price_cycle">
+            <h2>Cycle</h2>
+            <!-- Div for display highlighted price -->
+            <div id="cycle_display_price">
+                <h2>$100</h2>
+                <span>/Month</span>
```

Comment:
price and month should be in the same line.

Reply:
Done